.github/workflows/ci-pipeline.yml

```
@@ -79,8 +112,20 @@ jobs:
           push: true
           tags: ${{ secrets.DOCKER_HUB_USERNAME }}/todo-app:prod
           target: production
+      -
+        name: Terraform init
+        run: cd terraform/modules/prod && terraform init
```

Minor, but I do prefer the `working-directory` argument over a `cd && ...` approach personally:
```suggestion
        run: terraform init
        working-directory: terraform/modules/prod
```